zilencer/auth.py

```diff
@@ -61,7 +61,7 @@ def rate_limit_remote_server(
     try:
         RateLimitedRemoteZulipServer(remote_server, domain=domain).rate_limit_request(request)
     except RateLimitedError as e:
-        logger.warning("Remote server %s exceeded rate limits on domain %s", remote_server, domain)
+        logger.warning("Remote server %r exceeded rate limits on domain %s", remote_server, domain)
```

Comment:
(I haven’t gone through all our logging statements for changes like this, just the ones expected by the test suite. The message might actually be clearer with `%s` anyway.)

Reply:
Yeah `%s` is probably better here.